src/dmd/root/rmem.d

```diff
@@ -147,6 +148,8 @@ extern (C++) struct Mem
 
         static void disableGC() nothrow @nogc
         {
+            __gshared string[] disable_options = [ "gcopt=disable:1" ];
+            rt_options = disable_options;
```

Comment:
Should this discard all other options?

Reply:
It only replaces the currently embedded empty options, command line options still have higher precedence.
If the frontend is used as a library, you might want to use different options (though probably with the GC enabled anyway), so maybe it should be changed outside of rmem.d, e.g. mars.d?